src/resources/Fields/FieldsInterfaces.ts

```diff
@@ -32,3 +34,17 @@ export interface ListFieldsParams extends Paginated {
     sortBy?: string;
     type?: FieldTypes;
 }
+
+export interface FieldListingFilters {
+    facet?: FacetOrSortStatus;
+    name?: string;
+    origin?: FieldOrigin;
+    sort?: FacetOrSortStatus;
+    type?: FieldTypes;
+}
+
+export interface FieldListingOptions extends Paginated {
+    filter?: FieldListingFilters;
```

Comment:
Did you mean `filters`? ([doc](https://docs.coveo.com/en/8/api-reference/field-api#tag/Fields/operation/getFieldsUsingPOST_6))